examples/matmul.py

```diff
@@ -24,10 +24,48 @@ def matmul(x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
     return out
 
 
+@helion.kernel(static_shapes=True)
+def matmul_with_bias(
```

Comment:
Rather than copy and pasting the kernel, could we use the approach in https://github.com/pytorch-labs/helion/blob/main/examples/template_via_closure.py

Reply:
updated to pass in bias-add via epilogue func